repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/dataset.ftl

```diff
@@ -78,6 +78,12 @@
                 <h3><@fmt.message key="headers.resources"/></h3>
                 <ul data-uipath="ps.dataset.resources">
                     <#list dataset.files as attachment>
+                        <li class="attachment">
+                            <a title="${attachment.text}" href="<@hst.link hippobean=attachment.resource/>">${attachment.text}</a>;
+                            <span class="fileSize">size: <@formatFileSize bytesCount=attachment.resource.length/></span>
+                        </li>
+                    </#list>
+                    <#list dataset.filesOld as attachment>
```

Comment:
Reference to this section needs to be added to the JIRA ticket referencing all bits of code that need removing once the data upgrade scripts are run (see comment in the review of `Publication.java`).